elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +109,18 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView.bind(this).call();
```

Comment:
I wouldn't have guessed you'd need to bind `this` anywhere for these calls because this function itself is not called as an event listener.  Can you describe where you were running into context issues?